bin/admin/bootstrap-gce.sh

```diff
@@ -31,4 +31,8 @@ echo $enablestring >> $acctfile
 echo "install conda with Python 3 and 2 ..."
 $HOME/opt/gce/bin/admin/install-conda.sh
 
+# get ready to use SOLVCON
+echo "building SOLVCON..."
+$HOME/opt/gce/bin/admin/install-solvcon.sh $acctfile
```

Comment:
I think SOLVCON installation should not be included in bootstrapping.  We may provide a push-button command like ``bin/admin/install-solvcon.sh`` script, but executing it during bootstrapping may be too much.

In a cluster, a head node may not need to have the code installed.